### DISCUSSION TITLE
Community: known reuses

### DISCUSSION THREAD

**Discussion by raphodn** [openfoodfacts/open-prices#562] [General]
### List

- wiki: https://wiki.openfoodfacts.org/Project:Open-Prices#Known_re-uses
- on the website: https://prices.openfoodfacts.org/community

### Know a reuse ?

Share it here.
Or better, make a PR by editing the [reuses.json](https://github.com/openfoodfacts/open-prices-frontend/blob/master/src/data/reuses.json)

### Template

```
- name: 
- description: 
- background_image_url:
- url: 
- code_url: 
- date: 
- author: 
- display: true/false
```

### Related issues

- https://github.com/openfoodfacts/open-prices/issues/532
- https://github.com/openfoodfacts/open-prices-frontend/pull/1032
- https://github.com/openfoodfacts/open-prices/discussions/590

**Comment by raphodn** [comment]
### Location map

#### Info

- name: Map (Shiny)
- description: A world map of all the contributions in Open Prices
- background_image_url: 
- url: https://dmayaux.shinyapps.io/open_prices"
- code_url: 
- date: 2024
- author: Damien
- display: true

#### Screenshot

![Screenshot from 2024-11-12 22-45-06](https://github.com/user-attachments/assets/f87e57a8-1e04-44c0-b48f-cf88c0d515ff)

**Comment by raphodn** [comment]
### Price of carrots

#### Info

#### Screenshot

![Screenshot from 2024-11-12 22-43-35](https://github.com/user-attachments/assets/95734480-d862-4156-a8c7-cae83c747f77)

**Comment by raphodn** [comment]
### Do you want your receipt?

#### Info

Compare prices of 2 shops

#### Screenshot

![Screenshot from 2024-11-12 22-35-04](https://github.com/user-attachments/assets/643bf97c-818e-4138-9eb8-dfaf02470a81)

**Reply by raphodn** [reply]
Not online anymore...

**Reply by raphodn** [reply]
But we've built an alternative internally!
https://prices.openfoodfacts.org/locations/compare

**Comment by raphodn** [comment]
### Price graph

#### Info

#### Screenshot

![Screenshot from 2024-11-12 22-45-23](https://github.com/user-attachments/assets/82e982a3-7200-4f3d-8507-ccaef1d8babe)

**Comment by raphodn** [comment]
## Prixdle

### Info

http://prixdle.com/

Discovered at FOSDEM by @alexfauquette 

### Screenshot

![image](https://github.com/user-attachments/assets/88185fa5-3a07-4f73-a3cd-9d662d070196)

**Reply by raphodn** [reply]
FOSDEM

![IMG_20250201_130607](https://github.com/user-attachments/assets/3112350a-9568-4c7f-a1ed-f3a5983de55c)

**Comment by raphodn** [comment]
## Nutrition Dashboard

https://dietoptimization.com/

@albertaillet

**Comment by TTalex** [comment]
Shared by grunskis on slack:

> Hey all!
> I'm building an app to manage digital receipts (aka eBons) from German supermarkets (currently Lidl, Kaufland, Rewe, Edeka & DM are supported). Some weeks ago I learned about the Open Prices project. I find the project mission great :blush: Yesterday I released a new version of my app with an export feature to Open Prices. 
> I think it's pretty cool how quickly prices can be pushed to Open Prices. Oh, maybe the most interesting feature, besides how quickly prices can be pushed, is the automatic redaction of sensitive information from the receipts. I didn't include it in the video, but that's the part I spent the most time on :laughing: Here're  redacted proofs I pushed from the app: https://prices.openfoodfacts.org/users/ebonitoapp/proofs
> Let me know if you try it out and have any feedback. The app is called eBonito and is available only in the German App Store (simply b/c I support only a stores in Germany). Here's the homepage with some more info: https://ebonito.de/
